doc/classes/VisualServer.xml

```diff
@@ -17,6 +17,11 @@
 	<tutorials>
 		<link>https://docs.godotengine.org/en/latest/tutorials/optimization/using_servers.html</link>
 	</tutorials>
+	<members>
```

Comment:
Note that `<members>` comes after `<methods>` normally. `godot --doctool .` should fix it.

Reply:
Thanks, that fixed it but also removed some other stuff, so I'll just push what's related to this PR.

Reply:
Fixed